database/things/cobol.pldb

```diff
@@ -19,6 +19,7 @@ features
   MOVE ABC to XYZ  *> This is a comment.
  hasPrintDebugging true
  hasSemanticIndentation false
+ hasCaseInsensitiveIdentifiers true
```

Comment:
Not according to https://rosettacode.org/wiki/Case-sensitivity_of_identifiers#COBOL

Reply:
Hi @celtic-coder , it is not case sensitive according to https://devops.com/the-beauty-of-the-cobol-programming-language-v2/#:~:text=FAST%20FACT%3A%20Is%20COBOL%20a,named%20ws%2Dfirst%2Dname.

Reply:
From what I've seen, the Rosetta Code example may not be completely correct. Other examples, such as https://www.geeksforgeeks.org/variables-in-cobol/ seem to indicate that case does not matter. Let's go with case-insensitive.